packages/lesswrong/lib/collections/posts/custom_fields.js

```diff
@@ -1023,6 +1025,23 @@ Posts.addField([
       group: formGroups.adminOptions,
     }
   },
+  
+  {
+    fieldName: 'LWTableOfContents',
+    fieldSchema: {
+      type: Object,
+      optional: true,
+      viewableBy: ['guests'],
+      resolveAs: {
+        fieldName: "tableOfContents",
+        type: GraphQLJSON,
+        resolver: async (document, args, options) => {
+          return Utils.extractTableOfContents(document.htmlBody);
+        },
+        addOriginalField: true,
```

Comment:
Why would you want to add the original field here? If I am not confused, there should never be data in there. I would unify the two variable names, and then remove this.

Reply:
Me imitating existing resolvers without fully understanding what all the fields do, IIRC.